packages/client/wallets/smart-wallet/src/blockchain/wallets/EVMSmartWallet.ts

```diff
@@ -112,4 +125,39 @@ export class EVMSmartWallet {
     public async nfts() {
         return this.crossmintService.fetchNFTs(this.address, this.chain);
     }
+
+    public async sendTransaction<
+        const TAbi extends Abi | readonly unknown[],
+        TFunctionName extends ContractFunctionName<TAbi, "nonpayable" | "payable"> = ContractFunctionName<
+            TAbi,
+            "nonpayable" | "payable"
+        >,
+        TArgs extends ContractFunctionArgs<TAbi, "nonpayable" | "payable", TFunctionName> = ContractFunctionArgs<
+            TAbi,
+            "nonpayable" | "payable",
+            TFunctionName
+        >
+    >({
+        address,
+        abi,
+        functionName,
+        args,
```

Comment:
These 4 args are identical to the viem `writeContract` method which is used to call a function on a contract